leasing/models/rent.py

```diff
@@ -979,7 +979,13 @@ class LeaseBasisOfRent(ArchivableModel, TimeStampedSafeDeleteModel):
     In Finnish: Vuokranperusteet
     """
     lease = models.ForeignKey('leasing.Lease', verbose_name=_("Lease"), related_name='basis_of_rents',
-                              on_delete=models.PROTECT)
+                              on_delete=models.PROTECT, null=True, blank=True)
```

Comment:
Is there a need to change the `lease` relationship to not required?

Reply:
This is witch fails the lock test. But if this is not optional then it needs to be set from the client, i did not like that option. Ill think about this if there is any other alternative.

Reply:
If there is lease and parent defined then the returned json does look funny also as the object would be returned multiple times.